components/global/CxpFooter.vue

```diff
@@ -1,15 +1,17 @@
 <template>
   <div class="bg-gray p-4 mt-6 md:mt-10">
-    <p class="text-center text-blue-light">Copyright © 2019 ColtXP</p>
+    <p class="text-center text-blue-light">Copyright © {{year}} ColtXP</p>
   </div>
 </template>
 
 <script>
-  export default {
-    name: 'CxpFooter'
+export default {
+  name: "CxpFooter",
+  data() {
+    return { year: new Date().getFullYear() };
```

Comment:
I think this makes more sense as a computed property.

Reply:
Ah interesting.